apps/ensadmin/src/components/connections/default-connections-list.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import { CopyButton } from "@/components/copy-button";
+import { DropdownMenuItem, DropdownMenuLabel } from "@/components/ui/dropdown-menu";
+import { cn } from "@/lib/utils";
+
+interface Connection {
+  url: string;
+  isDefault: boolean;
```

Comment:
```suggestion
  fromServerLibrary: boolean;
```

For background, please see related comments